openpype/plugins/publish/collect_sequence_frame_data.py

```diff
@@ -0,0 +1,47 @@
+import pyblish.api
+import clique
+
+
+class CollectSequenceFrameData(pyblish.api.InstancePlugin):
+    """Collect Sequence Frame Data
+    """
+
+    order = pyblish.api.CollectorOrder + 0.2
+    label = "Collect Sequence Frame Data"
+    families = ["plate", "pointcache",
+                "vdbcache", "online",
+                "render"]
+    hosts = ["traypublisher"]
+
+    def process(self, instance):
+        frame_data = self.get_frame_data_from_repre_sequence(instance)
```

Comment:
Looking at your function implementation below this can return `None`

```suggestion
        frame_data = self.get_frame_data_from_repre_sequence(instance)
        if not frame_data:
            return
```